### PR TITLE
Separate avi vip network

### PR DESCRIPTION
**Pull request by iXinqi**

### CODE REVIEW

File: addons/packages/ako-operator/1.5.0/README.md [hunk 1]
```diff
@@ -29,15 +29,20 @@ The following configuration values can be set to customize the ako-operator inst
 | `avi_password` | Required | Avi controller password. |
 | `avi_cloud_name` | Required | the configured cloud name on the Avi controller. |
 | `avi_service_engine_group` | Required | the group name of Service Engine that's to be used by the set of AKO Deployments. |
+| `avi_management_cluster_service_engine_group` | Required | the group name of Service Engine that's to be used by the AKO in management cluster. |
```
Comment (XudongLiuHarold) [inline addons/packages/ako-operator/1.5.0/README.md:32]: This is good catch, thanks @iXinqi

File: addons/packages/ako-operator/1.5.0/README.md [hunk 2]
```diff
@@ -29,15 +29,20 @@ The following configuration values can be set to customize the ako-operator inst
...
+| `avi_cp_network` | Required | describes the Control Plane Networks of the workload cluster. |
+| `avi_cp_network_cidr` | Required | describes the Control Plane Networks of the workload cluster. |
 | `avi_ca_data_b64` | Required | Avi controller credential. |
 | `avi_labels` | Optional | Label used to select Clusters. The Clusters that are selected by this will be the ones affected by this AKODeploymentConfig. |
 | `avi_cni_plugin` | Optional | describes which cni plugin cluster is using. AKO supported CNI: antrea,calico,canal,flannel,openshift and ncp |
 | `avi_disable_static_route_sync` | Optional | describes ako should sync static routing or not. |
 | `avi_control_plane_ha_provider` | Required | describes whether Avi provides control plane HA service or not. |
 | `avi_management_cluster_vip_network_name` | Required | describes the data network name of the management cluster AKO will be deployed with. |
 | `avi_management_cluster_vip_network_cidr` | Required | describes the data network cidr of the management cluster AKO will be deployed with. |
+| `avi_management_cluster_cp_vip_network_name` | Required | describes the name of the management cluster control plane network. |
```
Comment (XudongLiuHarold) [inline addons/packages/ako-operator/1.5.0/README.md:44]: Can we avoid use `cp`? I think we can use `control_plane`, don't be afraid the name is long, we need it to be clear.

File: addons/packages/ako-operator/1.5.0/bundle/config/upstream/akooperator/static.yaml [hunk 3]
```diff
@@ -335,6 +346,7 @@ spec:
             - adminCredentialRef
             - certificateAuthorityRef
             - cloudName
+            - controlPlaneNetwork
```
Comment (XudongLiuHarold) [inline addons/packages/ako-operator/1.5.0/bundle/config/upstream/akooperator/static.yaml:349]: Same here, the field should be optional.

File: addons/packages/ako-operator/1.5.0/bundle/config/values.yaml [hunk 4]
```diff
@@ -18,15 +18,20 @@ akoOperator:
...
+    avi_cp_network: ""
+    avi_cp_network_cidr: ""
     avi_ca_data_b64: ""
     avi_labels: '""'
     avi_cni_plugin: antrea
     avi_disable_static_route_sync: false
     avi_control_plane_ha_provider: false
     avi_management_cluster_vip_network_name: ""
     avi_management_cluster_vip_network_cidr: ""
+    avi_management_cluster_cp_vip_network_name: ""
```
Comment (XudongLiuHarold) [inline addons/packages/ako-operator/1.5.0/bundle/config/values.yaml:33]: `cp` is not clear about what it actually meaning, we can change it to `control_plane`

**Review by XudongLiuHarold — CHANGES_REQUESTED** [review]
Generally, it looks good, needs some minor changes.

**Comment by iXinqi** [comment]
Will merge this PR after the network separation feature completed.

**Pull request closed without merging**